src/privatesend/privatesend.h

```diff
@@ -43,7 +43,6 @@ enum PoolMessage {
     ERR_MN_LIST,
     ERR_MODE,
     ERR_NON_STANDARD_PUBKEY,
-    ERR_NOT_A_MN, // not used
```

Comment:
Can't do this, enum is going to be broken. Alternatively, you could change next line to `ERR_QUEUE_FULL = 14,` but I would prefer to keep it as is instead.